vsts/StartEmulator.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+echo 'Listing available android sdks for installation'
+$ANDROID_HOME/tools/bin/sdkmanager --list | grep system-images
+
+emulatorImage='system-images;android-28;google_apis;x86_64'
```

Comment:
are we testing just a single API version, or the mix and max that we claim to support?

Reply:
We only ever tested API 28 on App center, and this PR doesn't change that. In theory though, we could run the same tests on two or more android API versions

Reply:
The list operation on line 4 actually prints out all the alternatives to the ```system-images;android-28;google_apis;x86_64``` that can be installed

Reply:
> In theory though, we could run the same tests on two or more android API versions

Is the effort small enough to add to this PR directly?

Reply:
Sure, but we haven't had a requirement like that yet. I could see that being a separate PR, if we need it at all